src/main/resources/com/fortify/plugin/jenkins/FortifyPlugin/config.jelly

```diff
@@ -291,11 +291,11 @@
             <f:textbox/>
           </f:entry>
           <f:entry field="timeout" title="Timeout" help="/plugin/fortify/help-timeout.html">
```

Comment:
Isn't it necessary to show unit like s or ms?

Reply:
You can check out setting helper (question mark icon on the right) where the information about the unit of the field is specified.